### ISSUE TITLE
Becomes default player for mp4 files

### ISSUE THREAD

**Issue by eonist** [sindresorhus/gifski-app#40]
Quicktime should be the designated app to open .mp4. Manually setting gifski to open all mp4 is ok. Setting it by default on install, is a bit overreaching IMO.

Best gif app since Drop2Gif. Gave it 5 stars on the App Store. ✌️

**Comment by sindresorhus** [comment]
This was fixed in Gifski 1.3.0 (Latest version on the App Store): https://github.com/sindresorhus/gifski-app/commit/a238e0383a5d1c7f7f10ca3731f4a3062f0d12f3

**Comment by sindresorhus** [comment]
> Best gif app since Drop2Gif. Gave it 5 stars on the App Store. ✌️

🤩

**Comment by eonist** [comment]
OK, I just downloaded the app 10 minutes ago. Maybe the Norwegian App Store is out of date or something. I tried to look for the bug in issue search but couldn't find it, so thought I would mention it.

**Comment by sindresorhus** [comment]
Hmm...

> Determines how Launch Services ranks this app among the apps that declare themselves editors or viewers of files of this type. The possible values are: Owner (this app is the primary creator of files of this type), Default (this app is an opener of files of this type; this value is also used if no rank is specified), Alternate (this app is a secondary viewer of files of this type), and None (this app is never selected to open files of this type, but it accepts drops of files of this type). Launch Services uses the value of LSHandlerRank to determine the app to use to open files of this type. The order of precedence is: Owner, Default, Alternate. This key is available in macOS 10.5 and later and iOS 3.0 and later. - https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/TP40009249-101685-TPXREF107

I assumed `Alternative` would make it still prefer the system apps, but it seems that's not true, so I'm going change it to `None`.

**Comment by sindresorhus** [comment]
Ok, looking at other video conversion apps, it seems all of them declare themselves as a `Viewer` and not `Editor`, so I guess that's the issue.


---

```xml
<dict>
	<key>CFBundleTypeExtensions</key>
	<array>
		<string>mp4</string>
	</array>
	<key>CFBundleTypeName</key>
	<string>MPEG-4 FIle</string>
	<key>CFBundleTypeRole</key>
	<string>Viewer</string>
</dict>
````

```xml
<dict>
	<key>CFBundleTypeMIMETypes</key>
	<array/>
	<key>CFBundleTypeName</key>
	<string>GIF Brewery 3 Supported Videos</string>
	<key>CFBundleTypeRole</key>
	<string>Viewer</string>
	<key>LSItemContentTypes</key>
	<array>
		<string>public.dv-movie</string>
		<string>public.3gpp</string>
		<string>public.3gpp2</string>
		<string>com.apple.m4v-video</string>
		<string>public.mpeg-2-video</string>
		<string>public.mpeg-4</string>
		<string>com.apple.quicktime-movie</string>
	</array>
	<key>LSTypeIsPackage</key>
	<integer>0</integer>
	<key>NSDocumentClass</key>
	<string>HRDocument</string>
</dict>
```

```xml
<dict>
	<key>CFBundleTypeName</key>
	<string>Media File</string>
	<key>CFBundleTypeRole</key>
	<string>Viewer</string>
	<key>LSItemContentTypes</key>
	<array>
		<string>public.data</string>
		<string>public.audio</string>
		<string>public.movie</string>
		<string>public.mpeg</string>
		<string>com.apple.quicktime-movie</string>
		<string>public.avi</string>
		<string>public.mpeg-4</string>
		<string>public.3gpp2</string>
		<string>public.3gpp</string>
	</array>
</dict>
```
